CMakeLists.txt

```diff
@@ -240,29 +240,26 @@ if(LMMS_BUILD_APPLE)
 endif()
 find_package(Perl)
 
-IF(WANT_LV2)
-	IF(PKG_CONFIG_FOUND)
-		PKG_CHECK_MODULES(LV2 lv2)
-		PKG_CHECK_MODULES(LILV lilv-0)
-	ENDIF()
-	IF(NOT LV2_FOUND AND NOT LILV_FOUND)
-		UNSET(LV2_FOUND CACHE)
-		UNSET(LILV_FOUND CACHE)
-		FIND_PACKAGE(LV2 CONFIG)
-		FIND_PACKAGE(LILV CONFIG)
-		IF(LILV_FOUND)
-			SET(LILV_LIBRARIES "lilv::lilv")
-		ENDIF()
-	ENDIF()
-	IF(LV2_FOUND AND LILV_FOUND)
-		SET(LMMS_HAVE_LV2 TRUE)
-		SET(STATUS_LV2 "OK")
-	ELSE()
-		SET(STATUS_LV2 "not found, install it or set PKG_CONFIG_PATH appropriately")
-	ENDIF()
-ELSE(WANT_LV2)
-	SET(STATUS_LV2 "not built as requested")
-ENDIF(WANT_LV2)
+if(WANT_LV2)
+	pkg_check_modules(LV2 lv2)
+
+	find_package(Lilv)
+	if(Lilv_FOUND)
+		set(LILV_LIBRARIES Lilv::lilv)
```

Comment:
Hmm.. minor but I think this should technically be `set(LILV_LIBRARIES "${Lilv_LIBRARY}")` as it's a required arg for [FindLilv.cmake](https://github.com/LMMS/lmms/pull/7702/files#diff-32c07265e6cb1f3f7c4e5750b4e2822d484e5d2bff6b66ecae558180252a2ac9).

Reply:
It looks like we may make this same mistake for SDL but that's out of scope for this PR.

Reply:
... and yes, adding our own variable `FOO_LIBRARIES` rather than using `Foo_LIBRARY` is redundant and should be fixed down the road.  In the future, we should probably forward-declare the libs we want to link lmms against and just pick it up in src later, but this change is better left for a cmake refactor.

Reply:
Other dependencies with a `Foo_LIBRARIES` variable set it to their CMake target rather than their library file path. I'm worried it won't work if I make this change.

Reply:
... to add more to this point, the redundant checking and adding of the include directories also seems like something we can consolidate in the future.  I believe that nearly every time we check for an include or a lib they're both added to src/, so we we should be able to feed two bird with one scone. 🐦

Reply:
> Other dependencies with a `Foo_LIBRARIES` variable set it to their CMake target rather than their library file path. I'm worried it won't work if I make this change.

You might be right.  I can dive into this more, but will be AFK for a while.  Sorry for the delay.

Reply:
From what I can tell, either format is acceptable.  This is what lmms links against on macOS.  Notice that it's a mix of `Foo::Foo`, `/usr/lib/foo` and `-framework Foo`.

```
Qt5::Core
Qt5::Gui
Qt5::Widgets
Qt5::Xml
Qt5::Svg
SDL2::SDL2
weakjack
suil-0
lilv-0
/opt/homebrew/lib/libfftw3f.dylib
SampleRate::samplerate
SndFile::sndfile
-framework CoreMIDI
-framework CoreFoundation
portaudio
mp3lame::mp3lame
Vorbis::vorbisenc
Vorbis::vorbisfile
```

Reply:
I agree, `Lilv::lilv` is preferred though.

Reply:
Ok, I guess we've added these manually in the same fashion, but in src/CMakeLists.txt.  So your approach is more consistent with how we do it now, just not in the same location as other deps which is why I missed it.

```cmake
if(LMMS_HAVE_MP3LAME)
	list(APPEND EXTRA_LIBRARIES mp3lame::mp3lame)
endif()

if(LMMS_HAVE_OGGVORBIS)
	list(APPEND EXTRA_LIBRARIES Vorbis::vorbisenc Vorbis::vorbisfile)
endif()
```

Reply:
Leaving unresolved for final comments, but please consider this resolved.